### DISCUSSION TITLE
Taking over the Vulnerator Project

### DISCUSSION THREAD

**Discussion by ebarlowjr2** [Vulnerator/Vulnerator#230] [General]
How many users still use the last updated version of vulnerator? Looking at possibly modernizing it with a new UI and updated features.  I dont see any current builds past 2019

**Comment by StevenMcGann** [comment]
I do.  Currently the old version still works.  I hope to see this project continue as Vulnerator is a go-to tool in my toolbox.

**Reply by ebarlowjr2** [reply]
If you know of others that use it have them comment or check in. I noticed a lot of users aren't developers of course and I'm looking to work with some teams to get this updated with new features and UI.

**Comment by Jkklein** [comment]
I still use it as much as I can.  I used it at least weekly and it would be
great to have it updated.

**Comment by CyberSecDef** [comment]
For what it's worth, I made a very similar  scans to report type generator a few years ago.  Feel free to use it...or use it for inspiration on any new dev efforts.

https://github.com/CyberSecDef/scans2reports

**Comment by RBremmer** [comment]
Several of our customers prefer the output of the Vulnerator tool for comparison and analysis.
